src/main/pg/adc.c

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
In general, can I propose that we call the directory `parameter_groups`? Using abbreviations that are not commonly used is a bit confusing for users not already familiar with the code.

Reply:
there is already `fc\` . PG is used all over the code (and spelling it out would be tedious)

Reply:
I think 'fc' is a lot better understood in the RC / multirotor field than PG...

Reply:
We already have `cms`, `fc/`, `io/`, `msp/`, `rx/` and `vcp/`. Whereas I'm generally not in favour of abbreviations, I think it is fine for the actual directory itself.

Reply:
I don't see that as a good excuse to introduce more abbreviation. On the other hand I don't see a downside to _not_ using abbreviations for directories (which are crucial for users to find their way when navigating the source).

Reply:
It's also a question of consistency. All PG macros and IDs are prefixed `PG_`. All PG functions are prefixed `pg`. You're not going to get very far in betaflight if you don't know PG stands for Parameter Group.

Reply:
All except for the file names...